### PR TITLE
Bring pipeline-cli and nr-pipeline-ext into nrdk code base (js -> ts)

### LINKED ISSUE #11: Move nr-pipeline-ext into nrdk code base

### PR DESCRIPTION
**Pull request by DerekRoberts**
Resolves #11 and https://github.com/BCDevOps/nrdk/issues/81

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ import {
   GitOperation,
   CONST,
 } from '@bcgov/nr-pipeline-ext'
-import {OpenShiftClient, OpenShiftClientX} from '@bcgov/pipeline-cli'
+import {OpenShiftClientX} from '@bcgov/pipeline-cli'
```
Comment (cvarjao) [inline src/index.ts:10]: Import needs to be updated to a local reference

Comment (cvarjao) [inline src/index.ts:10]: Import needs to be updated to a local reference

**Review by cvarjao — CHANGES_REQUESTED** [review]

File: src/pipeline-cli/is-openshift-list.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import isPlainObject from 'lodash.isplainobject'
+
+export function isPlainObj(object: any): boolean {
```
Comment (mbystedt) [inline src/pipeline-cli/is-openshift-list.ts:3]: Shoud be: isOpenshiftList?

**Review by mbystedt — APPROVED** [review]

File: src/pipeline-cli/is-openshift-list.ts (hunk 2 above)
Comment (DerekRoberts) [reply src/pipeline-cli/is-openshift-list.ts:3]: @mbystedt Yes, but I was told not to take any liberties bringing in pipeline-cli and nr-pipeline-ext.  That would be a definite fix on the next pass.  Thanks for reviewing!  :)

Original:
https://github.com/BCDevOps/pipeline-cli/blob/master/lib/isOpenShiftList.js

**Pull request merged**